cfgov/jinja2/v1/_includes/molecules/rss-feed.html

```diff
@@ -9,8 +9,20 @@
    Creates a block containing RSS feed markup.
 
    ========================================================================== #}
-
+{% macro render( url ) %}
 <div class="block block__sub" data-qa-hook="rss-subscribe-section">
```

Comment:
`block block__sub` should very likely not appear in the molecule, but instead should appear where this molecule is included.

Reply:
Can you elaborate, Ans? When I collapsed `rss.html` and `rss-feed.html` into one file, I included this `div`, but is that problematic from an atomic standpoint or...? Andy and I aren't sure. 😄

Reply:
Yeah I'd see it as problematic from an atomic standpoint. 
See this diagram:

<img width="824" alt="Screen Shot 2019-03-08 at 5 00 27 PM" src="https://user-images.githubusercontent.com/704760/54058397-ba356380-41c3-11e9-80b4-5025009573d0.png">

Component A works across all kinds of contexts because the margin is set by its parent container. Component B runs into problems because each new context requires customizations of the component.

Reply:
@anselmbradford do you know what `data-qa-hook="rss-subscribe-section"` does? I see [some tests](https://github.com/cfpb/cfgov-refresh/search?q=rss-subscribe-section&unscoped_q=rss-subscribe-section) that refer to it, but does that do anything? Right now that's duplicate both outside and inside of the element. It would simplify to be able to remove it.

Reply:
The idea was to have an attribute dedicated to the element that the tests were looking for so that the CSS class could change and the tests would still pass. So depending on how the tests are written I'm pretty sure that should go on the component itself, not its container.